src/client/score.js

```diff
@@ -40,4 +40,26 @@ function calculScoreTotal(frameTable){
     return sum;
 }
 
-module.exports = calculScoreTotal;
\ No newline at end of file
+module.exports = calculScoreTotal;
+
+function calcFrame10(frame){
```

Comment:
What if the frame is null ? undefined ?
What if C1 / C2 / C3 are equal to null ? maybe you should get the 3 values first, test and add tests on null

Reply:
I can't check all data from the start because of  case like "waiting thrid thow" the value will be C1 / C2 / Null. But C1+ C2 can be a valid score

Reply:
So you must add test like if(C1!=null) before usine their value.